internal/misc/bullets.go

```diff
@@ -0,0 +1,10 @@
+package misc
```

Comment:
misc/utils/helpers/common packages are root of all evil 👿 
This function here depends on **slacks's** emotes, therefore should be in slack package.

The same for exclamation file.

Reply:
Yeah, I figured it's `misc` but in the scope of the app, not like "Global `misc`! Hey everybody, let's use it all over the place!". I think I see the flaw in this thinking, now.